docs/docs/getting-started/installation/aws.md

```diff
@@ -243,14 +296,19 @@ do
 done
 ```
 
-Delete the gateways and the routes in the VPC
+Delete the gateways and the routes in the VPC:
 
 ```sh
 for gateway in $(aws ec2 describe-internet-gateways --filters Name=attachment.vpc-id,Values=${vpc_id} --query 'InternetGateways[].InternetGatewayId' --output text)
 do
     aws ec2 detach-internet-gateway --internet-gateway-id ${gateway} --vpc-id ${vpc_id}
     aws ec2 delete-internet-gateway --internet-gateway-id ${gateway}
 done
+```
+
+Delete the route tables (expect that the command will fail for the default route table, but still you shouldn't have any problem deleting the VPC in the next step):
```

Comment:
`expect that the command will fail for the default route table` if it always fail we should reword like "the command will always fail for the default route table but it should still...."